### PR TITLE
update badge-url

### PR DESCRIPTION
**Pull request by torbsorb**

### CODE REVIEW

**Comment by trym-b** [comment]
I get this when I try to click on the badge:
image

**Comment by torbsorb** [comment]
I get that for zivid-python too.. however, the badge does at least seem to point to something that builds correctly. I got the updated URL information from https://dev.azure.com/zivid-devops/_apis/projects/python-samples and Definition from 
`HTTP GET https://dev.azure.com/zivid-devops/python-samples/_apis/build/definitions?api-version=5.1`:
```
StatusCode        : 200
StatusDescription : OK
Content           : {"count":1,"value":[{"_links":{"self":{"href":"https://dev.azure.com/zivid-devops/701a6042-3865-4412-9f7f-78b846c1a406/_apis/build/Definitions/3?revision=1"},"web":{"href":"https://dev.azure.com/zivid...
RawContent        : HTTP/1.1 200 OK
                    Pragma: no-cache
                    X-TFS-ProcessId: e8aba81c-b0fc-4834-aa1a-062ab5b5de1b
                    Strict-Transport-Security: max-age=31536000; includeSubDomains
                    ActivityId: 4377a495-c722-4731-bb9c-585cd4b6e5...
Forms             :
Headers           : {[Pragma, no-cache], [X-TFS-ProcessId, e8aba81c-b0fc-4834-aa1a-062ab5b5de1b], [Strict-Transport-Security, max-age=31536000; includeSubDomains], [ActivityId, 4377a495-c722-4731-bb9c-585cd4b6e59c]...}
Images            : {}
InputFields       : {}
Links             : {}
ParsedHtml        :
RawContentLength  : 1853
```

File: README.md [hunk 1]
```diff
@@ -74,8 +74,8 @@ If you need assistance with using Zivid cameras, visit our Knowledge Base at [ht
 ## Licence
 Zivid Samples are distributed under the [BSD license](source/LICENSE).
 
-[ci-badge]: https://img.shields.io/azure-devops/build/zivid-devops/376f5fda-ba80-4d6c-aaaa-cbcd5e0ad6c0/2/master.svg
-[ci-url]: https://dev.azure.com/zivid-devops/python-samples/_build/latest?definitionId=2&branchName=master
+[ci-badge]: https://img.shields.io/azure-devops/build/zivid-devops/701a6042-3865-4412-9f7f-78b846c1a406/3
```
Comment (trym-b) [inline README.md:77]: shouldn't it be `/master.svg` in the end?

Comment (torbsorb) [reply README.md:77]: it's not in cpp

**Comment by SatjaSivcev** [comment]
What is the status of this PR?

**Pull request merged**